architecture/additional_concepts/authorization.adoc

```diff
@@ -476,13 +476,15 @@ when you upgrade {product-title}.
 
 === Applying Custom Roles and Permissions
 
-To add or update custom roles and permissions, it is strongly recommended to use
+To update custom roles and permissions, it is strongly recommended to use
 the following command:
 
 ----
-# oc auth reconcile -f FILE
+# oc auth reconcile -f <file>
```

Comment:
Do we have an example of the roles file that we can link to? If so, consider adding a link to it, like:

~~~
----
# oc auth reconcile -f <file> <1>
----
<1> `<file>` is the absolute path to the xref:../../link.adoc#heading[roles and permissions] to apply.
~~~

Reply:
@kalexand-rh I did some looking around to find an example of a yaml file for creating cluster roles. PTAL. Thanks!